### PR TITLE
Students form validation and service tests.

### PR DESCRIPTION
**Pull request by MichaelValkiv**

### CODE REVIEW

File: src/app/admin-panel/admin-panel/students/students.component.scss [hunk 1]
```diff
@@ -32,7 +32,7 @@
 }
 .studentsForm ::ng-deep input
 {
-  width: 100%;
+  width: 100% !important;
```
Comment (dmitryy90) [inline src/app/admin-panel/admin-panel/students/students.component.scss]: why is this in the ng-deep?

Comment (dmitryy90) [reply src/app/admin-panel/admin-panel/students/students.component.scss]: input must be acceptable

Comment (MichaelValkiv) [reply src/app/admin-panel/admin-panel/students/students.component.scss]: why i use !important, or what?

Comment (MichaelValkiv) [reply src/app/admin-panel/admin-panel/students/students.component.scss]: Removed !importent.

Comment (dmitryy90) [reply src/app/admin-panel/admin-panel/students/students.component.scss]: the question was: why do you use ng-deep if this input is the component template?

Comment (MichaelValkiv) [reply src/app/admin-panel/admin-panel/students/students.component.scss]: without ng-deep, i can't set style for p-inputMask and p-calendar

**Pull request merged**